cpp/src/groupby/sort/aggregate.cpp

```diff
@@ -94,6 +94,17 @@ void aggregrate_result_functor::operator()<aggregation::SUM>(aggregation const&
                      get_grouped_values(), helper.num_groups(), helper.group_labels(), stream, mr));
 };
 
+template <>
+void aggregrate_result_functor::operator()<aggregation::PRODUCT>(aggregation const& agg)
+{
+  if (cache.has_result(col_idx, agg)) return;
+
+  cache.add_result(col_idx,
```

Comment:
Is `cache` a local, a parameter, or a member variable? If it is a member, then we should add a prefix like `_cache` to differentiate it.

Reply:
All except `agg` are member variables. `_` doesn't help much in differentiating.